enterprise/cmd/frontend/internal/batches/resolvers/changeset_test.go

```diff
@@ -210,6 +210,7 @@ func TestChangesetResolver(t *testing.T) {
 		ExternalServiceType:   "github",
 		ExternalID:            "98765",
 		ExternalForkNamespace: "user",
+		ExternalForkName:      "user",
```

Comment:
Hahaha cool fork name. 😛 Not that it really matters but maybe:

```suggestion
		ExternalForkName:      "my-fork",
```